src/olympia/reviewers/forms.py

```diff
@@ -312,6 +315,18 @@ def validate_review_attachment(value):
                     'file {extensions}.'.format(extensions=valid_extensions_string)
                 )
             )
+        if value.size >= settings.MAX_ZIP_UNCOMPRESSED_SIZE:
```

Comment:
This setting has a specific meaning that is subtly different from maximum file size, as it's the maximum size of the zip files _contents_ once unzipped. We might change that value in the future, so let's use a new setting instead.

Call that new setting `MAX_UPLOAD_SIZE`, stick that in `lib/settings_base.py` with a comment explaining that this is our own setting and not Django's and it needs to be implemented by relevant forms. This would make https://github.com/mozilla/addons/issues/14835 a nice follow-up for you to work on after if you have the bandwidth.

Note that dev/stage/prod are currently configured with `client_max_body_size 200m` so the python code won't even see requests trying to upload files over 200 MB anyway, but we could bump it to 250 MB to give a little bit of leeway (we won't accept requests too big to avoid DOS attacks and similar with large requests).